hr_contract_amendment/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=====================
+HR CONTRACT AMENDMENT
```

Comment:
@EBII Good module, but still need to be improved a little bit...Use title formating here, no upper...